tidy3d/components/device/simulation.py

```diff
@@ -840,3 +863,54 @@ def from_scene(cls, scene: Scene, **kwargs) -> HeatSimulation:
             medium=scene.medium,
             **kwargs,
         )
+
+    def _get_simulation_types(self) -> list[DeviceSimulationType]:
```

Comment:
I'm in favor of moving this to backend. The only issue I have right now is that this function is called in `plot_scene_specs` so that electric specs are plotted when the simulation type is CONDUCTION. This isn't a huge problem since we can always add an additional parameter to `plot_scene_specs`. I guess my question would be, where do we move this to so that it is accessible from heat_pipeline and heat_mesh/heat_setup?